app/services/encryptor.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Encryptor
+  KEY = ActiveSupport::KeyGenerator.new(
+    ENV.fetch('SECRET_KEY_BASE'),
```

Comment:
Would it be more idiomatic to use Rails.application.secret_key_base here?

Reply:
@starswan good point 👍